site/en/docs/crux/history-api/index.md

```diff
@@ -0,0 +1,617 @@
+
+{% Aside %}
+Unlike the daily API, the CrUX History API does not have data aggregated against Effective Connection Type (ECT).
+{% endAside %}
+
+### Metric
+
+Metrics are expressed in a histogram array, which represents the percent of users that experienced a metric with that value proportionally to all.
+
+A simple three bin histogram for an example metric looks like this:
+
+```json
+{
+  "histogramTimeseries": [
+    {
+      "start": 0,
+      "end": 2500,
+      "densities": [0.9190, 0.9203, 0.9194, 0.9195, 0.9183, 0.9187]
+    },
+    {
+      "start": 2500,
+      "end": 4000,
+      "densities": [0.0521, 0.0513, 0.0518, 0.0518, 0.0526, 0.0527]
+    },
+    {
+      "start": 4000,
+      "densities": [0.0288, 0.0282, 0.0286, 0.0285, 0.0290, 0.0285]
+    }
+  ],
+}
+```
+
+This data indicates that 91.2% of users experience the example metric value between 0ms and 2,500ms for the first collection period in the history, followed by 92.0%, 91.9%, and so on. The units of the metric are not contained in this histogram, in this case we will assume milliseconds.
+
+Additionally, 5.2% of users experience the example metric value between 2,500ms and 4,000ms in the first collection period in the history, and 2.8% of users experience a value greater than 4,000ms in the first collection period in the history.
+
+Metrics will also contain percentiles that can be useful for additional analysis.
+
+```json
+{
+  "percentilesTimeseries": {
+    "p75s": [1362, 1352, 1344, 1356, 1366, 1377]
+  },
+}
+```
+
+These percentiles can show specific metric values at the given percentile for that metric. They are based on the full set of available data and not the final binned data, so they do not necessarily match an interpolated percentile that is based on the final binned histogram.
+
+{% Aside %}
+Note: The values for each percentile are synthetically derived, it does not imply that any user actually experienced the value indicated, only that some percentage of users experienced a metric value that was less than the value given.
+{% endAside %}
+
+#### Metric value types
+
+As the CrUX History API uses the same metric value types, you can reference [the daily CrUX API metric value types documentation](../api/#metric-value-types) for more details.
+
+#### Metric eligibility
+
+Based on the [eligibility criteria](../methodology/#eligibility) an origin or URL may only be eligible for some of the collection periods covered by the CrUX History API. In these cases the CrUX History API will return `"NaN"` for the `histogramTimeseries` densities and `null` for the `percentilesTimeseries` for the collection periods which have no eligible data. For example, if the second period did not have any eligible data, this would show as:
+
+```json
+{
+  "histogramTimeseries": [
+    {
+      "start": 0,
+      "end": 2500,
+      "densities": [0.9190, "NaN", 0.9194, 0.9195, 0.9183, 0.9187]
```

Comment:
Unclear why this is `"NaN"` unlike the `null` values in `percentilesTimeseries`. (we can discuss in the team chat)